### ISSUE TITLE
Add/investigate value readout for AccessibleNumberTweaker

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/sun#356]
Currently it doesn't have a readout for the current value.

(potentially) desired for Area Model.

**Comment by zepumph** [comment]
To be clear. We may not want to publish Area Model with any interactive alerts, because we are only implementing static descriptions, so it may be good to have a way to navigate the PDOM and see what the current value of the spinner is.

**Comment by zepumph** [comment]
@jessegreenberg what do you think?

**Comment by terracoda** [comment]
@zepumph, I am not sure there is a way to silence values on spinners and sliders. You likely know this, but the values themselves are not alerts, but just the current value.

**Comment by zepumph** [comment]
What is the best way to proceed with this issue then @terracoda? Is the `aria-valuenow=4` enough? Are we done here?

![image](https://user-images.githubusercontent.com/6856943/40860898-731e74d0-6593-11e8-93db-524a70458168.png)

**Comment by terracoda** [comment]
@zepumph, this is looking good.
Is the context for this issue that the read outs were a bit non-sensical in the first go around?

I think the actual pdom might want to read more like this:

Factors, button, expanded
Change dimensions of rectangle.
Height, stepper, <aria-valuetext: 4> times  Width, stepper, <aria-valuetext: 1>

Or 

Factors, button, expanded
Area Height, stepper, <aria-valuetext: 4> times  Area Width, stepper, <aria-valuetext: 1>
Change dimension factors of rectangle, height times width.

Or eventually when there was a dynamic math phrase it might be like this:

Factors, button, expanded
4 x 1 (read as "four times one")
Area Height, stepper, <aria-valuetext: 4> times  Area Width, stepper, <aria-valuetext: 1>
Change dimension factors of rectangle, height times width.

I'll have to check with @amanda-phet for context.

**Comment by zepumph** [comment]
So it looks like the `AccessibleNumberTweaker` is acting accordingly, back to you @jonathanolson to work with @terracoda and @amanda-phet about designing the descriptions.

**Comment by jessegreenberg** [comment]
Discussed on 7/17/18 - The common work for this issue is complete. Any remaining work will be sim specific, lets create an issue in area-model-common to discuss what the remaining sim specific readout for this input should be.
